docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -1,23 +1,155 @@
+
+.. image:: ../../../../utils/images/semantic_seg_example.png
+  :width: 600
+  :alt: image uploaded from this `source <https://arxiv.org/abs/1912.03183>`_
+
+
+We solve this task by utilizing `FCN Head <https://arxiv.org/pdf/1411.4038.pdf>`_ with implementation from `MMSegmentation <https://mmsegmentation.readthedocs.io/en/latest/_modules/mmseg/models/decode_heads/fcn_head.html>`_ on the multi-level image features obtained by the feature extractor backbone (`Lite-HRNet <https://arxiv.org/abs/2104.06403>`_).
+For the supervised training we use the following algorithms components:
+
+.. _semantic_segmentation_supervised_pipeline:
+
+- ``Augmentations``: Besides basic augmentations like random flip, random rotate and random crop, we use mixing images technique with different `photometric distortions <https://mmsegmentation.readthedocs.io/en/latest/api.html#mmseg.datasets.pipelines.PhotoMetricDistortion>`_.
+
+- ``Optimizer``: We use `Adam <https://arxiv.org/abs/1412.6980>`_ optimizer with weight decay set to zero and gradient clipping with maximum quadratic norm equals to 40.
+
+- ``Learning rate schedule``: For scheduling training process we use **ReduceLROnPlataeu** with linear learning rate warmup for 100 iterations. This method monitors a target metric (in our case we use metric on the validation set) and if no improvement is seen for a 'patience' number of epochs, the learning rate is reduced.
+
+- ``Loss function``: We use standart `Cross Entropy Loss <https://en.wikipedia.org/wiki/Cross_entropy>`_  to train a model.
+
+- Additionally, we use the **early stopping** to add adaptability to the training pipeline and prevent overfitting.
+
 **************
 Dataset Format
 **************
 
-To be added soon
+For the dataset handling inside OTX we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_.
+
+At this end we support `ADE20K <https://openvinotoolkit.github.io/datumaro/docs/formats/ade20k2020/>`_, `Cityscapes <https://openvinotoolkit.github.io/datumaro/docs/formats/cityscapes/>`__, `Pascal VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_ and `Common Semantic Segmentation <https://openvinotoolkit.github.io/datumaro/docs/formats/common_semantic_segmentation/>`_ data formats.
+If you organized supported dataset format, starting training will be very simple. We just need to pass a path to the root folder and desired model template to start training:
+
+.. code-block::
+
+    $ otx {train, optimize} <model_template> --train-data-root <path_to_data_root>
+
+
+.. note::
+
+    OTX will recognize train and validation splits automatically. If you want to emphasize the validation dataset root path, you can simply add the ``--val-data-root <path_to_val_data_root>`` option.
+
+    Please, refer to our :doc:`dedicated tutorial <../../../tutorials/base/how_to_train/semantic_segmentation>` how to train, validate and optimize semantic segmentation model for more details.
+
+
```

Comment:
Before saying that we support the custom format in seg, I hope we add the description line that we basically support VOC and cityscapes dataset original format in Datumaro.

Reply:
I'm not fully understand what you mean by that, but I described data formats supported by Datumaro in the 36-41 lines. Even gave a CLI command line how to run training